src/Tools/DynamoFeatureFlags/FeatureFlagsClient.cs

```diff
@@ -137,16 +137,10 @@ private static string GetConfigurationItem(String key,string configFilePath)
         {
             try
             {
-                var doc = new XmlDocument();
-                doc.Load(configFilePath);
-                if (doc != null)
-                {
-                    XmlNode node = doc.SelectSingleNode("//appSettings");
-
-                    XmlElement value = (XmlElement)node.SelectSingleNode(string.Format("//add[@key='{0}']", key));
-                    return value.Attributes["value"].Value;
-                }
-                return null;
+                IConfigurationRoot config = new ConfigurationBuilder()
+                    .AddJsonFile(System.IO.Path.Combine(new System.IO.FileInfo(configFilePath).DirectoryName, "appsettings.json"), false, true)
+                    .Build();
+                return config[key];
```

Comment:
Can we test this code ?

Reply:
I will keep this change in stash, so we can apply it when necessary